core/src/main/java/org/apache/carbondata/core/datastore/filesystem/LocalCarbonFile.java

```diff
@@ -368,6 +372,8 @@ public DataOutputStream getDataOutputStream(String path, FileFactory.FileType fi
       outputStream = new SnappyOutputStream(new FileOutputStream(path));
     } else if ("LZ4".equalsIgnoreCase(compressor)) {
       outputStream = new LZ4BlockOutputStream(new FileOutputStream(path));
+    } else if ("ZSTD".equalsIgnoreCase(compressor)) {
+      outputStream = new ZstdOutputStream(new FileOutputStream(path), 1);
```

Comment:
what is the meaning of 1?

Reply:
compress level 1.  We have test using default level 3 which can get better compress ratio but cost more time. As for sort temp file which is not used for storage, level 1 is cost-effective.